src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs

```diff
@@ -16,7 +16,7 @@ internal abstract class FixSomeCodeAction : CodeAction
         private static readonly HashSet<string> s_predefinedCodeFixProviderNames = GetPredefinedCodeFixProviderNames();
 
         internal readonly FixAllState FixAllState;
-        private readonly bool _showPreviewChangesDialog;
+        private bool _showPreviewChangesDialog;
```

Comment:
:memo: This field is never written by production code. It could preserve the `readonly` modifier if the `TestAccessor.ShowPreviewChangesDialog` property had access to `Unsafe.AsRef<T>(in T)`, which would allow test code to rewrite the value for the purpose of a specific test.